clang/test/SemaSYCL/sycl-no-rdc.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clang_cc1 -fsycl-is-device -verify -fsyntax-only -fsycl-allow-func-ptr -fno-gpu-rdc %s
+SYCL_EXTERNAL void syclExternal();
+
+SYCL_EXTERNAL void notSyclExternal() {}
+
+template <typename KernelName, typename KernelType>
+[[clang::sycl_kernel]] void kernel_single_task(const KernelType& kernelFunc) { // #kernelSingleTask
```

Comment:
Current guidelines are to use the mock header file and the interface from there, to more closely mimic the user code.   For example, see sycl-device-template-diag.cpp

Reply:
fixed, thanks

Reply:
FYI, I don't see the FE changes.

Reply:
sorry i didnt push that commit yet, i should have held off on commenting until i pushed my bad